tools/accuracy_checker/accuracy_checker/adapters/detection.py

```diff
@@ -705,3 +705,148 @@ def _find_output(predictions):
                 'If you need to use another layer, please specify it explicitly'
             )
         return filter_outputs[0]
+
+
+class FaceBoxesSSDDecoder(Adapter):
+    """
+    Class for converting output of FaceBoxes SSD models to DetectionPrediction representation
+    """
+    __provider__ = 'faceboxes_ssd_decoder'
+
+    def validate_config(self):
+        super().validate_config(on_extra_argument=ConfigValidator.ERROR_ON_EXTRA_ARGUMENT)
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'scores_out': StringField(description="Scores output layer name."),
+            'boxes_out': StringField(description="Boxes output layer name."),
+        })
+
+        return parameters
+
+    def configure(self):
+        self.scores_out = self.get_value_from_config('scores_out')
+        self.boxes_out = self.get_value_from_config('boxes_out')
+
+        # Set default values
+        self.min_sizes = [[32, 64, 128], [256], [512]]
+        self.steps = [32, 64, 128]
+        self.variance = [0.1, 0.2]
+        self.confidence_threshold = 0.05
+        self.nms_threshold = 0.3
+        self.keep_top_k = 750
+
+    @staticmethod
+    def prior_boxes(feature_maps, sizes, image_size, steps):
+        anchors = []
+        for k, f in enumerate(feature_maps):
+            min_sizes = sizes[k]
+            for i, j in itertools.product(range(f[0]), range(f[1])):
+                for min_size in min_sizes:
+                    s_kx = min_size / image_size[1]
+                    s_ky = min_size / image_size[0]
+                    if min_size == 32:
```

Comment:
not sure that cycle necessary here, because for all `min_size` we have different behaviour

Reply:
@eaidova please see updated version